google-cloud-spanner/lib/google/cloud/spanner/errors.rb

```diff
@@ -20,6 +20,16 @@ module Cloud
     module Spanner
       ##
       # @private
+      # # CommitError
+      #
+      # Signal that the Transaction has been committed.
```

Comment:
The preferred usage for simple scenarios should be to not call `Transaction#commit`, but just to let the block complete.  My understanding is that `Transaction#commit` should only be called in case of some conditional logic or similar scenario.